composer.json

```diff
@@ -51,10 +51,7 @@
     },
     "bin": ["bin/doctrine-dbal"],
     "config": {
-        "sort-packages": true,
-        "platform": {
-            "php": "7.3.0"
```

Comment:
W/o the platform fixed, all dependency updates will have to be made on the lowest supported version. Otherwise, there's a risk of locking the dependencies that are not compatible with it.

It might be better to unset the platform version on CI before the update but have it committed in the repo.

Reply:
When supporting this range of versions, you need to "composer update" anyways to get to the right packages. So the solution is probably rather to remove composer.lock in this step. With this setting in, you cannot get the code running locally with 7.1 and 7.2

Reply:
But without this setting, the one performing a dependency update will have to use PHP 7.1. Otherwise, they will learn that the new resolution doesn't work with PHP 7.1 only on CI. Is there a Composer CLI switch to pass `config.platform.php` at runtime?

Reply:
This point is not relevant anymore since ce3421e961cea783afb0ddfb3339478ca09a5d5a composer.lock is not committed anymore and the Ci always fetches the latest packeges matching all constraints.